docs/api/proposed_endpoints.md

```diff
@@ -356,3 +356,177 @@ Status: 200 OK
     "posted_time": null
 }
 ```
+
+# Proposals
+
+## Create Proposal
+
+```
+POST /api/proposal
+```
+
+**Parameters**
+
+Name               | Data type     | Description
+-------------------|---------------|---------------------
+id                 | UUID          | Unique ID for the proposal
+title              | text          | Title of proposal
+speaker            | text          | Speaker for the talk
+kind               | text          | Type of proposal like talk, dev sprint, workshop
+level              | text          | Level of proposal beginner, intermediate, advanced
+duration           | text          | Duration of talk, sprint or workshop 
+abstract           | text          | Abstract of the proposal
+description        | text          | Description of the proposal
+submitted_at       | datetime      | Time of submission of proposal
+approved_at        | datetime      | Time of approval
+modified_at        | datetime      | Time of modification
+proposed_time      | datetime      | Proposed time of event
```

Comment:
What is `proposed_time` referring to? When does this come into play?

Reply:
I thought there should be a  time slot at which the  speaker prefers to present the talk.  Is it correct? Or the review committee gives any slot of their choice?

Reply:
I would rather say this as `scheduled_time` which would be set once the proposal is accepted and the user would be given chance to state their preference of timing. But this all would be offline and is out of the scope of this product for now.